querydsl-spatial/src/main/java/com/querydsl/spatial/GeometryPaths.java

```diff
@@ -25,6 +25,8 @@
 import org.geolatte.geom.Polygon;
 
 /**
+ * {@code GeometryPaths} provides factory methods for {@link GeometryExpression} creation
```

Comment:
Shouldn't `{@link GeometryExpression}` be `{@link GeometryPath}`?
I only see `Path` subtypes.